composer.json

```diff
@@ -36,6 +36,8 @@
         "symfony/http-client": "^4.4|^5.0"
     },
     "require-dev": {
+        "aws/aws-sdk-php": "^3.172",
```

Comment:
I had to put the entire AWS SDK to be able to Assume Role because [AsyncAws does not support STS Regional Endpoints](https://github.com/async-aws/aws/issues/1061) (ping @Nyholm)

Reply:
Let’s generate that operation then :)